### DISCUSSION TITLE
Custom Layout Slots - Modified Date

### DISCUSSION THREAD

**Discussion by fug4zi** [microsoft-search/pnp-modern-search#2732] [Q&A] [answered]
Hi,

i try to change the Slot "Date" from "Created" to "Modified" but the field is always empty.

Is the name different to modified?
Under the [Datasource FAQ Page](https://microsoft-search.github.io/pnp-modern-search/usage/search-results/data-sources/#sharepoint-search) i did not find any "modified" property

But in the [Slots FAQ Page](https://microsoft-search.github.io/pnp-modern-search/usage/search-results/slots/#builtin-slots) there is written: A date associated to the item. Can be for instance, the created date, modified date or publishing date.

Thanks for clarify

**Comment by wobba** [comment, marked as answer]
The property is named `LastModifiedDate`. And ensure it's in the list of retrieved properties as well.

**Reply by fug4zi** [reply]
i only found the LastModifiedTime field in the properties list
![image](https://user-images.githubusercontent.com/1719274/216283076-a282a34f-3436-45df-8e2d-c53394f91ab7.png)

But with this i get what i wanted
![image](https://user-images.githubusercontent.com/1719274/216283226-051fa143-20d3-4aba-a88a-6dd72fb5beee.png)

Thanks for your fast reply

**Reply by wobba** [reply]
My bad.. it's the Time one. Was too quick when writing :)
